template/metadata.json

```diff
@@ -0,0 +1 @@
+{"file": "script.sh", "name": "template", "description": "A template for managed scripts which lists master machines of the logged in cluster.", "author": "supreeth7", "allowedGroups": ["SREP"], "rbac": {"roles": [{"namespace": "openshift-machine-api", "rules": [{"verbs": ["get", "list", "watch"], "apiGroups": ["machine.openshift.io"], "resources": ["machines"]}]}]}, "language": "bash"}
```

Comment:
This file is not needed as it is a temp file for validation, could you help to remove it?